src/App.js

```diff
@@ -1,66 +1,25 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
 import "./App.css";
 import Card from "react-bootstrap/Card";
 import CardDeck from "react-bootstrap/CardDeck";
 import Accordion from "react-bootstrap/Accordion";
 import Button from "react-bootstrap/Button";
 import { getData } from "./actions/getData";
-import { connect } from "react-redux";
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      content: [{ Positive: [] }, { Negative: [] }, { Neutral: [] }],
-      PoisitivePage: 1,
-      NegativePage: 1,
-      NeutralPage: 1
-    };
-    this.renderCardtext = this.renderCardtext.bind(this);
-    this.renderPagenumbers = this.renderPagenumbers.bind(this);
-    this.handleClick = this.handleClick.bind(this);
-  }
-
-  componentDidMount() {
-    let Sentiment = require("sentiment");
-    let sentiment = new Sentiment();
-    this.props.dispatch(getData()).then(() => {
-      this.props.apiData.forEach((content, i) => {
-        if (content.review_body + content.review_headline) {
-          let res = sentiment.analyze(
-            content.review_body + content.review_headline
-          );
-          let temp, category, stateCopy;
-          stateCopy = Object.assign({}, this.state);
-          switch (true) {
-            case res.score < 0 ||
-              content.star_rating === "1" ||
-              content.star_rating === "2":
-              category = 1;
-              temp = "Negative";
-              break;
-            case res.score > 0 || content.star_rating === "5":
-              category = 0;
-              temp = "Positive";
-              break;
-            case res.score === 0:
-              category = 2;
-              temp = "Neutral";
-              break;
-            default:
-              break;
-          }
-          stateCopy.content[category][temp].push({
-            Title: content.product_title,
-            review_body: content.review_body,
-            review_headline: content.review_headline,
-            Date: content.review_date,
-            Sentiment_Score: res.score
-          });
-          this.setState(stateCopy);
-        }
-      });
-    });
+function App() {
+  const [contentVal, setContent] = useState([
+    { Positive: [] },
+    { Negative: [] },
+    { Neutral: [] },
+  ]);
+  const [PoisitivePage, setPoisitivePage] = useState(1);
+  const [NegativePage, setNegativePage] = useState(1);
+  const [NeutralPage, setNeutralPage] = useState(1);
+  const apiData = useSelector((state) => state.getDataReducer.apiData);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getData());
 
     /* Data format
```

Comment:
Add comments at the top of the components for 3 formats:
1. raw data from amazon format
2. data passed through sentiment module format
3. local state array format